packages/test-kit/src/with-feature.ts

```diff
@@ -337,6 +337,9 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}) {
 
             const featurePage = await browserContext.newPage();
 
+            navigationOptions = navigationOptions || {};
+            navigationOptions.waitUntil = navigationOptions.waitUntil || 'load';
```

Comment:
the default of `waitUntil` is 'load', maybe 'domcontentloaded'?
https://playwright.dev/docs/api/class-page#page-goto